stories/DropdownItem.js

```diff
@@ -0,0 +1,337 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+import DropdownItem from '../src/components/DropdownItem';
+import defaultTheme from '../src/themes/DefaultTheme';
+import darkTheme from '../src/themes/DarkTheme';
+
+import {
+  DownloadIcon,
+  PadlockUnlockIcon,
+  TrashBinIcon,
+} from '../src/components/icons';
+
+storiesOf('DropdownItem', module)
+  .addWithJSX('Dropdown item with min props', () => {
+    function onItemClick(e) {
+      console.log('click');
+    }
+
+    return (
+      <>
+        <DropdownItem
+          title="Dropdown item with min props"
+          onClick={onItemClick}
```

Comment:
We don't need to pass anything to `onClick` prop because the default is an empty function ;)